aspnetcore/tutorials/signalr.md

```diff
@@ -241,10 +245,18 @@ The SignalR server must be configured to pass SignalR requests to SignalR.
 
 ## Next steps
 
+In this tutorial, you learned how to:
+
+> [!div class="checklist"]
+> * Create a web app project.
+> * Add the SignalR client library.
+> * Create a SignalR hub.
+> * Configure the project to use SignalR.
+> * Add code that uses the hub to send messages from any client to all connected clients.
+
 If you want clients to connect to a SignalR app from different domains, you have to enable Cross-Origin Resource Sharing (CORS). For more information, see [Cross-origin resource sharing](xref:signalr/security?view=aspnetcore-2.1#cross-origin-resource-sharing).
```

Comment:
The version shouldn't be included in this doc link. Please fix that while you're in here.

Reply:
I missed that sentence; for MVC compliance the whole sentence should be removed.